addons/dialogue_manager/l10n/zh-cn.po

```diff
@@ -0,0 +1,403 @@
+msgstr "从 CSV 导入"
+
+msgid "confirm_close"
+msgstr "是否要保存到“{path}”？"
+
+msgid "confirm_close.save"
+msgstr "保存"
+
+msgid "confirm_close.discard"
+msgstr "不保存"
+
+msgid "buffer.save"
+msgstr "保存"
+
+msgid "buffer.save_as"
+msgstr "另存为……"
+
+msgid "buffer.close"
+msgstr "关闭"
+
+msgid "buffer.close_all"
+msgstr "全部关闭"
+
+msgid "buffer.close_other_files"
+msgstr "关闭其他文件"
+
+msgid "buffer.copy_file_path"
+msgstr "复制文件路径"
+
+msgid "buffer.show_in_filesystem"
+msgstr "在 Godot 侧边栏中显示"
+
+msgid "settings.revert_to_default_test_scene"
+msgstr ""
+
+msgid "settings.autoload"
+msgstr "Autoload"
+
+msgid "settings.path"
+msgstr "路径"
+
+msgid "settings.new_template"
+msgstr "新建文件时自动插入模板"
+
+msgid "settings.missing_keys"
+msgstr "将翻译键缺失视为错误"
+
+msgid "settings.missing_keys_hint"
+msgstr "如果你使用静态键，这将会帮助你寻找未添加至翻译文件的键。"
+
+msgid "settings.wrap_long_lines"
+msgstr "自动折行"
+
+msgid "settings.include_failed_responses"
+msgstr "在判断条件失败时仍显示选项"
+
+msgid "settings.ignore_missing_state_values"
+msgstr "忽略全局变量缺失错误（不建议）"
+
+msgid "settings.custom_test_scene"
+msgstr "自定义测试场景（必须继承自BaseDialogueTestScene）"
+
+msgid "settings.states_shortcuts"
+msgstr "全局变量映射"
+
+msgid "settings.states_message"
+msgstr "当一个 Autoload 在这里被勾选，他的所有成员会被映射为全局变量。"
+
+msgid "settings.states_hint"
+msgstr "比如，当你开启对于“Foo”的映射时，你可以将“Foo.bar”简写成“bar”。"
+
+msgid "n_of_n"
+msgstr "第{index}个，共{total}个"
+
+msgid "search.previous"
+msgstr "查找上一个"
+
+msgid "search.next"
+msgstr "查找下一个"
+
+msgid "search.match_case"
+msgstr "大小写敏感"
+
+msgid "search.toggle_replace"
+msgstr "替换"
+
+msgid "search.replace_with"
+msgstr "替换为"
+
+msgid "search.replace"
+msgstr "替换"
+
+msgid "search.replace_all"
+msgstr "全部替换"
+
+msgid "files_list.filter"
+msgstr "查找文件"
+
+msgid "titles_list.filter"
+msgstr "查找标题"
+
+msgid "errors.key_not_found"
+msgstr "键“{key}”未找到"
+
+msgid "errors.line_and_message"
+msgstr "第{line}行第{colume}列发生错误：{message}"
+
+msgid "errors_in_script"
+msgstr "你的脚本中存在错误。请修复错误，然后重试。"
+
+msgid "errors_with_build"
+msgstr "请先解决 Dialogue 中的错误。"
+
+msgid "errors.import_errors"
+msgstr "被引入的文件存在问题。"
+
+msgid "errors.already_imported"
+msgstr "文件已被引入。"
+
+msgid "errors.duplicate_import"
+msgstr "引入名不能重复。"
+
+msgid "errors.empty_title"
+msgstr "标题名不能为空。"
+
+msgid "errors.duplicate_title"
+msgstr "标题名不能重复。"
+
+msgid "errors.nested_title"
+msgstr "标题不能嵌套。"
+
+msgid "errors.invalid_title_string"
+msgstr "标题名非法。"
+
+msgid "errors.invalid_title_number"
+msgstr "标题不能以数字开始。"
+
+msgid "errors.unknown_title"
+msgstr "标题未定义。"
+
+msgid "errors.jump_to_invalid_title"
+msgstr "标题名非法。"
+
+msgid "errors.title_has_no_content"
+msgstr "目标标题为空。请替换为“=> END”。"
+
+msgid "errors.invalid_expression"
+msgstr "表达式非法。"
+
+msgid "errors.unexpected_condition"
+msgstr "未知条件。"
+
+msgid "errors.duplicate_id"
+msgstr "ID 重复。"
+
+msgid "errors.missing_id"
+msgstr "ID 不存在。"
+
+msgid "errors.invalid_indentation"
+msgstr "缩进非法。"
+
+msgid "errors.condition_has_no_content"
+msgstr "条件下方不能为空。"
+
+msgid "errors.incomplete_expression"
+msgstr ""
```

Comment:
This one is when you have an expression that doesn't quite look right:

![image](https://github.com/nathanhoad/godot_dialogue_manager/assets/78984/0a8b8fe8-3e89-4b18-96d2-523705af0036)